Packs/Alexa/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### New: Alexa Rank Indicator v2
+- Alexa v2 uses the Alexa API rank.
+- Domains which are not in Alexa DB, are considered "Unknown" instead of "Suspicious".
+- If the domain doesn't exist there is an error.
+- Default values changed:
```

Comment:
Please add the change list. Do check out other v2 or v3 integrations such as `VirusTotal v3` to see how they've done it.

Reply:
Added default values that was missing, Is this OK?